packages/common/src/schemas/documents/workflow/documents/schemas/ZZ.ts

```diff
@@ -86,6 +86,63 @@ export const getUniversalDocuments = (): TDocument[] => {
         issueDate: OptionalTypePastDate,
       }),
     },
+    {
+      category: 'proof_of_ownership',
+      type: 'business_utility_bill',
+      issuer: { country: 'ZZ' },
+      issuingVersion: 1,
+      version: 1,
+      propertiesSchema: Type.Object({}),
```

Comment:
Are you sure using Type.Object({}) won't fail non-empty properties objects?
```suggestion
      propertiesSchema: Type.Object({}),
```

Reply:
Fixed